src/containers/TweetPage.css

```diff
@@ -0,0 +1,15 @@
+.tweetHeader {
```

Comment:
it might be better to use bootstrap component

Reply:
Done